src/main/java/org/wso2/carbon/module/swiftiso20022/validation/common/ValidatorContext.java

```diff
@@ -27,6 +29,9 @@ public class ValidatorContext {
     private final String fieldDisplayName;
     private int fieldLength;
 
+    // to store any additional property to be used in custom validators
+    private Map<String, Object> additionalProperties;
```

Comment:
Can't we simply say properties here?

Reply:
Changed in commit 4e687a247135e6bc9d43274b4f9f97756cf64480